### PR TITLE
77654- Incomplete payment

### PR DESCRIPTION
**Pull request by will-hope98**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs [hunk 1]
```diff
@@ -148,6 +149,7 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
         }
 
         [HttpGet]
+        [SmallProducerSubmissionSubmitted(Order = 2)]
         [SmallProducerSubmissionContext]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs:153]: @will-hope98  add the order = 1 to the SmallProducerSubmissionContext

File: src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs [hunk 2]
```diff
@@ -744,6 +754,43 @@ public void DownloadSubmission_Get_ShouldHaveSmallProducerSubmissionContextAttri
...
         }
 
+        [Fact]
+        public void ProducerController_ShouldHaveAuthorizeRouteClaimsAttribute()
+        {
+            // Arrange
+            var typeInfo = typeof(ProducerController);
+
+            // Act
+            var attribute = (AuthorizeRouteClaimsAttribute)Attribute.GetCustomAttribute(typeInfo, typeof(AuthorizeRouteClaimsAttribute));
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs:764]: @will-hope98  similar test for the ProducerSubmissionController I think

File: src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs (hunk 1 above)
Comment (will-hope98) [reply src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs]: Done

File: src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs (hunk 2 above)
Comment (will-hope98) [reply src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs:764]: done

**Pull request merged**